mgmt/ProcessManager.cc

```diff
@@ -414,35 +414,35 @@ ProcessManager::handleMgmtMsgFromLM(MgmtMessageHdr *mh)
 {
   ink_assert(mh != nullptr);
 
-  char *data_raw = (char *)mh + sizeof(MgmtMessageHdr);
+  char *payload = static_cast<char *>(mh->payload());
```

Comment:
Should payload() perhaps just return a MemSpan?

Reply:
Yeah, that's reasonable.